terrapower/physics/neutronics/dragon/dragonWriter.py

```diff
@@ -26,37 +36,41 @@
 
 N_CHARS_ALLOWED_IN_LIB_NAME = 8
 
+MixtureNuclide = namedtuple(
```

Comment:
Consider moving down to where the DragonMixture is.

Also, can you document what types are expected on this.

Reply:
done and done